robot_specific_config/abb_irb14000_support/package.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" ?>
+<package format="3">
+  <name>abb_irb14000_support</name>
+  <version>0.0.0</version>
+
+  <description>
+    <p>
+      ROS-Industrial support for the ABB IRB 14000.
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for the ABB IRB 14000 robot.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in the
+      <em>ABB IRB 14000 Product specification</em>, document ID: <em>3HAC052982-001, Dec 2024</em>.
+      All urdfs / xacros are based on the default motion and joint velocity
+      limits, unless noted otherwise (ie: no support for high speed joints,
+      extended / limited motion ranges or other options).
+    </p>
+    <p>
+      Note: Maximum joint effort values do not correspond to real world limits
+      of the robot. The current values were chosen to accommodate simulation,
+      but are fictional values.
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+
+  <author>Gal Gorjup</author>
+  <maintainer email="gal@acumino.ai">Gal Gorjup</maintainer>
+
+  <license>Apache2</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <exec_depend>abb_resources</exec_depend>
+  <exec_depend>joint_state_publisher</exec_depend>
```

Comment:
```suggestion
  <exec_depend>joint_state_publisher</exec_depend>
  <exec_depend>joint_state_publisher_gui</exec_depend>
```

Reply:
Thanks! Actually it seems that `joint_state_publisher` isn't even used, so I'll just replace it with `joint_state_publisher_gui`.